.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -1200,6 +1372,19 @@ dynamodb_Scenario_GettingStartedMovies:
     - Delete the table.
   category: Scenarios
   languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/dynamodb/scenarios/DynamoDB_Basics
+          excerpts:
+            - description:
+              snippet_tags:
+                - DynamoDB.dotnetv3.dynamodb-basics.CreateTable
+                - DynamoDB.dotnetv3.dynamodb-basics.PutItem
+                - DynamoDB.dotnetv3.dynamodb-basics.UpdateItem
+                - DynamoDB.dotnetv3.dynamodb-basics.GetItem
+                - DynamoDB.dotnetv3.dynamodb-basics.DeleteItem
+                - DynamoDB.dotnetv3.dynamodb-basics.DeleteTableExample

```

Comment:
The scenario part is missing. This is just a list of the single-action wrapper functions. Fine to include these, but you also need to include the scenario code that puts them together.
Also, because C# is class-based, include the wrapping class declaration instead of just bare functions.
And don't forget to include some description to inform the reader what's what.
You're also missing functions like batch write, query, and scan. Compare to the epic to make sure you have all the parts listed in the spec.

Reply:
I found the missing functions in a different scenario. Include those here and remove them from the other scenario.